CMakeLists.txt

```
@@ -0,0 +1,227 @@
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
+
+project(KokkosTools CXX)
+
+# Include utilities
+include(cmake/utils.cmake)
+include(cmake/configure_tpls.cmake)
+
+# Set policies
+cmake_policy(SET CMP0111 NEW) # error if library not found
+
+# Disable in-source builds to prevent source tree corruption.
+if(CMAKE_SOURCE_DIR STREQUAL CMAKE_BINARY_DIR)
+  message(FATAL_ERROR "FATAL: In-source builds are not allowed. You should create a separate directory for build files.")
+endif()
+
+list(INSERT CMAKE_MODULE_PATH 0 ${PROJECT_SOURCE_DIR}/cmake)
+
+message(STATUS)
+message(STATUS Configuring Kokkos-Tools)
+message(STATUS)
```

I t looks weid to have a `message(STATUS)` without any message to print.
```suggestion
message(STATUS "Configuring Kokkos-Tools\n")
```